activiti-services/activiti-services-core-model/src/main/java/org/activiti/services/core/model/MetaBpmnModel.java

```diff
@@ -0,0 +1,37 @@
+package org.activiti.services.core.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.activiti.bpmn.model.BpmnModel;
+import org.activiti.services.core.model.json.converter.BpmnModelDeserializer;
+import org.activiti.services.core.model.json.converter.BpmnModelSerializer;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class MetaBpmnModel {
```

Comment:
@daisuke-yoshimoto this name is confusing... We have MetaBpmnModel, BpmnModel and ProcessModel. We need to unify all under just one model. BpmnModel make sense, but I don't know if we need that type anyway. It sounds to me that if we have something called BpmnModel just to represent the content (xml or json) as a string will be enough. So instead of having Serializers and Deserializers for our type we just have serialization and deserialization to a string.